apis/apis.json

```diff
@@ -238,8 +238,8 @@
     "description": "Guidelines for using Google Stackdriver Instrumentation Libraries for ASP.NET Core.",
     "tags": [ "CodeAnalysis", "Analyzers", "CSharp" ],
     "testDependencies": {
-      "Microsoft.AspNetCore.Mvc.Core": "1.0.2",
-      "Microsoft.AspNetCore.Routing": "1.0.2"
+      "Microsoft.AspNetCore.Mvc.Core": "1.0.4",
+      "Microsoft.AspNetCore.Routing": "1.0.4"
     }
   },
 
```

Comment:
Also update `Microsoft.AspNetCore.Http` in `Google.Cloud.Diagnostics.Common` below?

Reply:
I don't think we need to - certainly GitHub hasn't told me that there's a vulnerability there. (That's the only way I knew about this at all.)

I've only updated Routing in this case as it felt sensible to keep the two direct dependencies at the same version.